### DISCUSSION TITLE
How to check Lr_scheduler Parameter?

### DISCUSSION THREAD

**Discussion by WongiPark0628** [yandex-research/rtdl-revisiting-models#38] [Q&A] [answered]
I'm implement your review in my code. but I didn't find your lr_scheduler Parameter.

How to check lr_scheduler Parameters?

<img width="588" alt="image" src="https://user-images.githubusercontent.com/86551201/198876096-4bae173b-8602-4a1c-845e-f3e625df97fe.png">

**Comment by Yura52** [comment, marked as answer]
The function make_lr_schedule was not used in our experiments, we simply forgot to remove it from the codebase.

**Reply by WongiPark0628** [reply]
Thanks!
